jsapp/js/router/routerUtils.ts

```diff
@@ -8,11 +8,11 @@
  * of defined ROUTES.
  */
 
-import {
-  ROUTES,
-  PATHS,
-} from 'js/router/routerConstants';
+import {ROUTES, PATHS} from 'js/router/routerConstants';
 import {PROJECTS_ROUTES} from 'js/projects/routes';
+import session from '../stores/session';
+import {redirectDocument} from 'react-router';
+import {when} from 'jquery';
```

Comment:
Only one small issue: when I login after being redirected, instead of sending me to `/#/account/plans`, it sends me back to the login page. It looks like this should be [importing `when` from `'mobx'`](https://mobx.js.org/reactions.html#when) instead of jQuery.